### DISCUSSION TITLE
Test Text Message Could Not Be Sent... other messages work though

### DISCUSSION THREAD

**Discussion by Wanderwardr** [berrystephenw/OctoText#28] [Q&A]
After I installed and configured the plugin I sent a test message and got the following error message in OctoPrint.  Once I uploaded a file I got a notification on my phone from the email I set up, indicating that the plugin was working, but the test message function is not. 

![image](https://user-images.githubusercontent.com/81767894/113325033-b30eb000-92cc-11eb-9abb-68ee002d1328.png)

Update: The message that worked was a file uploaded notification.  Since I didn't get a print started notification, I tried disabling webcam images, and that seemed to work.  Any messages including an image do not seem to get through.

**Comment by berrystephenw** [comment]
Hi Wanderwardr,
Looking at the log it appears you are running into the flip/rotation bug. If you turn off the snapshots you won't have an issue, but as soon as it is enabled - and your camera is rotated there is a bug that I'm working on a fix for. Shouldn't take too long to verify. 

https://github.com/berrystephenw/OctoText/discussions/25#discussioncomment-558629

**Reply by Wanderwardr** [reply]
OK, Thanks for the answer and the quick reply!  I am looking forward to the update, but in the meantime having just the text is great.

**Reply by Wanderwardr** [reply]
Update: I oriented the camera so that no flip/rotation is necessary, and now I can get texts normally.

**Reply by berrystephenw** [reply]
I think I've fixed the bug. Version 0.1.6 should not have the issue.
